build_tools/benchmarks/run_benchmarks_on_linux.py

```diff
@@ -6,16 +6,140 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 """Runs all matched benchmark suites on a Linux device."""
 
+import subprocess
+import atexit
+import os
+import re
+import shutil
+import sys
+import tarfile
+from typing import Optional, Sequence
```

Comment:
Empty line after this.

Reply:
Added the empty line after `import tarfile`. Do we need to have a newline after `from typing import Optional, Sequence`?

Reply:
It's clearer that way given we are importing stuff from our own repo. :) But not a big deal. Your choice.

Reply:
I see. Done : )